backend/src/helpers/nativeLanguages.js

```diff
@@ -1,15 +1,17 @@
 const db = require("../models/nativeLanguage");
+const { get } = require("../routes/teachers");
```

Comment:
is this used anywhere?

Reply:
Don't think so — I didn't type that but it somehow is being reflected as an addition. weird